dracoon/src/main/java/ch/cyberduck/core/sds/SDSDirectS3UploadFeature.java

```diff
@@ -245,7 +245,8 @@ public void run() {
                         done.countDown();
                     }
                 }
-            }, new HostPreferences(session.getHost()).getLong("sds.upload.s3.status.period"), TimeUnit.MILLISECONDS);
+                                                     }, new HostPreferences(session.getHost()).getLong("sds.upload.s3.status.delay"),
```

Comment:
Indentation.
Eventually add `delay: ` and `period: ` explicit parameter labels in order to see what each line does.

Reply:
Not aware of such a syntax.

Reply:
Using `/* delay */` is probably not an option then.